packages/application-shell/src/components/redirect-to-project-create/redirect-to-project-create.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import { Spacings, Text, LinkButton } from '@commercetools-frontend/ui-kit';
+
+export class RedirectToProjectCreate extends React.Component {
+  static displayName = 'RedirectToProjectCreate';
+  static propTypes = {
+    servedByProxy: PropTypes.bool.isRequired,
+  };
+  constructor(props) {
+    super(props);
+
+    if (this.props.servedByProxy === true)
```

Comment:
I went with a constructor to avoid to use the deprecated `componentWillMount` will also avoiding a potential flash of something rendering when using `componentDidMount`. Open to suggestions (e.g. double check in `render`).